packages/beacon-node/src/api/impl/beacon/state/index.ts

```diff
@@ -290,7 +290,7 @@ export function getBeaconStateApi({
 
       return {
         data: {
-          validators: syncCommitteeCache.validatorIndices,
+          validators: new Array(...syncCommitteeCache.validatorIndices),
```

Comment:
is this intended?
```suggestion
          validators: new Array(syncCommitteeCache.validatorIndices),
```

Reply:
yes, the Array constructor is different than Uint8Array and friends, it requires an iterable or else it will store the param as a single item.